.pre-commit-config.yaml

```diff
@@ -56,6 +56,7 @@ repos:
     hooks:
       - id: prettier
         types_or: [yaml, markdown, html, css, javascript, json]
+        exclude: '^\.github/ISSUE_TEMPLATE/'
```

Comment:
This should not be necessary. We use these issue templates in our other repositories as well and they work just fine with prettier over there. Maybe worth investigating why this isn't working here.

Reply:
I noticed it was not in the other repositories, but without this my pre-commit calls always modify the templates...

I'll check if pre-commit is set up differently in this repository, or if there's any notable difference in the repo structure.

Reply:
Problem solved, I realized that the other repositories acutally use yml format rather than markdown. Should work now!